include/mysql.h

```diff
@@ -208,6 +208,7 @@ extern const char *SQLSTATE_UNKNOWN;
     /* MariaDB specific */
     MYSQL_PROGRESS_CALLBACK=5999,
     MYSQL_OPT_NONBLOCK,
+    MYSQL_OPT_IO_WAIT,
```

Comment:
New options (which are not supported by MySQL server should be prefixed with MariaDB and should be added to the end of enumeration.